products/containeranalysis/api.yaml

```diff
@@ -75,3 +133,113 @@ objects:
                   The human readable name of this Attestation Authority, for
                   example "qa".
                 required: true
+
+  - !ruby/object:Api::Resource
+    name: 'Occurrence'
+    base_url: projects/{{project}}/occurrences
+    self_link: projects/{{project}}/occurrences/{{name}}
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      An occurrence is an instance of a Note, or type of analysis that
+      can be done for a resource.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'https://cloud.google.com/container-analysis/'
+      api: 'https://cloud.google.com/container-analysis/api/reference/rest/'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: |
+          The name of the occurrence.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: resourceUri
+        description: |
+          Required. Immutable. A URI that represents the resource for which
+          the occurrence applies. For example,
+          https://gcr.io/project/image@sha256:123abc for a Docker image.
+        required: true
+        input: true
+      - !ruby/object:Api::Type::String
+        name: noteName
+        description: |
+          The analysis note associated with this occurrence, in the form of
+          projects/[PROVIDER_ID]/notes/[NOTE_ID]. This field can be used as a
+          filter in list requests.
+        required: true
+        input: true
+      - !ruby/object:Api::Type::String
+        name: kind
+        description: |
+          The note kind which explicitly denotes which of the occurrence
+          details are specified. This field can be used as a filter in list
+          requests.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: remediation
+        description: |
+          A description of actions that can be taken to remedy the note.
+      - !ruby/object:Api::Type::Time
+        name: createTime
```

Comment:
The request said all of the top-level fields so I ended up doing them, but I can take them out. Do you feel strongly that we shouldn't include them?

Reply:
Nah, I don't think it matters, just wondering why they were included here

Reply:
👍yeah I didn't feel strongly either - my (weak) argument would be that since this is meant to be a security measure/log of sorts times are good to have, and that if we decide to support a datasource they might be used as filters (and I will pretend that was the reason all along)